### ISSUE TITLE
ENOENT error on windows 7 when gitbook init

### ISSUE THREAD

**Issue by tanbro** [GitbookIO/gitbook#548]
my env:
* windows7 SP1 x86
* node v0.10.20
* gitbook 1.5.0

i installed the gitbook into the globally by:
```sh
npm install -g gitbook
```

and then executed `gitbook init` in my work dir:
```sh
cd /F/gitbooks
gitbook init
```
and the error is:
```
Error: ENOENT, open 'F:\gitbooks\SUMMARY.md'
```

**Comment by winniehell** [comment]
Does the file (SUMMARY.md) exist? I had the same issue before I noticed that you have to create the file manually. :wink:

**Comment by tanbro** [comment]
o, yes. it will be ok if create the `SUMMARY.md` file manually.

**Comment by eaigner** [comment]
Why did you close this issue? I don't think creating the `SUMMARY.md` manually is intended behavior. I have the same issue on OSX Yosemite

**Comment by eaigner** [comment]
At least it should print an error saying you have to create a `SUMMARY.md` file manually, not a cryptic `ENOENT` error.

**Comment by tanbro** [comment]
ok, @eaigner , let's reopen it

**Comment by SamyPesse** [comment]
@eaigner @tanbro why version of GitBook are you using?

If it's version 1.5.0, please test the 2.0.0 beta.

**Comment by winniehell** [comment]
Fixed for 2.0.0-beta.
`gitbook init` is now creating `SUMMARY.md`. :+1:

**Comment by tanbro** [comment]
nice, it is fixed!
